ci/build_docs.sh

```diff
@@ -22,9 +22,6 @@ if [ x"$DOC_BUILD" != x"" ]; then
     echo "Will build docs"
 
     source activate pandas
-    conda install -n pandas -c r r rpy2 --yes
-
-    time sudo apt-get $APT_ARGS install dvipng texlive-latex-base texlive-latex-extra
```

Comment:
so this will fail (I already tried it!) as the sudo line has to be run in the 'top' script (IOW ci/build-docs.sh) and not the spawned script (e.g. what we are running), so need to move *only* the sudo line back here